src/main/webapp/app/modules/administration/user-management/permission/permissions-title.tsx

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
-import { Text, View } from '@adobe/react-spectrum';
+import { Button, Flex, Text, View } from '@adobe/react-spectrum';
 import { Translate } from 'react-jhipster';
 
-const PermissionTitle = props => (
-  <View borderBottomWidth="thin" borderColor="default" height="size-700">
-    <p className="spectrum-Heading spectrum-Heading--sizeS" style={{padding: '15px 22px'}}>
-      <Translate contentKey="permissions.home.administrators">Administrators</Translate>
-    </p>
-  </View>
-);
+export interface IPermissionTitleProps {
+  handleSaveClick?: () => void;
+}
 
+const PermissionTitle = (props: IPermissionTitleProps) => {
+  const { handleSaveClick } = props;
+
+  const save = () => {
+    handleSaveClick();
+  };
+
+  return (
+    <View borderBottomWidth="thin" borderColor="default" height="size-700">
+      <Flex justifyContent="space-between">
+        <Flex justifyContent="center" alignItems="center">
+          <p className="spectrum-Heading spectrum-Heading--sizeS" style={{ padding: '15px 22px' }}>
+            <Translate contentKey="permissions.home.administrators">Administrators</Translate>
+          </p>
+        </Flex>
+        {handleSaveClick && (
+          <Flex alignItems="end">
+            <p className="spectrum-Heading spectrum-Heading--sizeS" style={{ padding: '15px 22px' }}>
+              <Button onPress={save} variant="cta">
+                <Translate contentKey="entity.action.save">Save</Translate>
+              </Button>
+            </p>
+          </Flex>
+        )}
+      </Flex>
+    </View>
```

Comment:
should not be included here separate component needs to be created and title component can be reused